org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/dom/SchemaLocation.java

```diff
@@ -44,11 +44,8 @@ public SchemaLocation(DOMAttr attr) {
 				break;
 			}
 			DOMNode valNode = attr.getNodeAttrValue();
-			// matcher matches 1 char ahead, but valNode's start is at beginning of "
-			int start = valNode.getStart() + locPairMatcher.start(2);
-			// matcher matches end correctly, but range needs to be one past end
-			// to highlight the end, and one to make up for "
-			int end = valNode.getStart() + locPairMatcher.end(2) + 2;
+			int start = valNode.getStart() + locPairMatcher.start(2) + 1;

```

Comment:
Please add a comments with String which uses | to understand what is the start and the end.